gravitee-apim-console-webui/src/services-ngx/notification-settings.service.ts

```diff
@@ -56,7 +56,7 @@ export class NotificationSettingsService {
 
   update(apiId: string, notificationId: string, notificationConfig: NotificationSettings): Observable<NotificationSettings> {
     return this.http.put<NotificationSettings>(
-      `${this.constants.env.baseURL}/apis/${apiId}/notificationsettings/${notificationId}`,
+      `${this.constants.env.baseURL}/apis/${apiId}/notificationsettings/${notificationId === 'PORTAL' ? '' : notificationId}`,
```

Comment:
I think I've already asked for it but it must have got lost. 
What do you think about renaming this service to `ApiNotificationSettingsService` like Application... & Environement...

Reply:
Done